things/openapi.yml

```diff
@@ -654,8 +654,7 @@ components:
       description: Unique thing identifier.
       in: path
       schema:
-        type: integer
-        minimum: 1
+        type: string
```

Comment:
Add here `format: uuid` according to https://swagger.io/docs/specification/data-models/data-types/